src/main/java/io/aiven/kafka/connect/common/output/OutputWriter.java

```diff
@@ -46,16 +48,30 @@
 
     private Boolean isClosed;
 
-    protected OutputWriter(final OutputStream outputStream, final OutputStreamWriter writer) {
+    protected final Map<String, String> externalConfiguration;
```

Comment:
This seems to be needed only for `ParquetOutputWriter`. Maybe keep it there?

Reply:
I think it would be useful for us in the future if we want to pass by additional config parameters for particular libraries